pkg/controller/datavolume-controller.go

```diff
@@ -2823,14 +2823,27 @@ func (r *DatavolumeReconciler) detachPvcDeleteDv(pvc *corev1.PersistentVolumeCla
 }
 
 func updatePvcOwnerRefs(pvc *corev1.PersistentVolumeClaim, dv *cdiv1.DataVolume) {
-	refs := pvc.OwnerReferences
-	for i, r := range refs {
-		if r.UID == dv.UID {
-			pvc.OwnerReferences = append(refs[:i], refs[i+1:]...)
+	var refs []metav1.OwnerReference
```

Comment:
can it be `foreignRefs` or `otherRefs`,

Reply:
It's not foreign or other, but the updated pvc ownerRefs, so I think the name is fine.

Reply:
ok, maybe you are right, 
but after line 2829 it only contains refs that do not have given DV as an owner, that made me think about name